modules/microshift-adding-containers-to-blueprint.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-embed-into-rpm-ostree.adoc
+
+:_content-type: PROCEDURE
+[id="adding-{product-title}-container-images_{context}"]
+= Adding the {product-title} container images
+
+You can embed {product-title}'s container images into the {op-system-ostree} images so that they are immediately available to the CRI-O container engine after booting. Embedded container images are not pulled over the network from a container registry. In Image Builder, a container image is embedded by adding a reference to it to the Image Builder blueprint.
+
+The following syntax must be used to add a configuration section to the blueprint file. You can then add your the container image to embed in the {op-system-ostree} image.
+
+.Example syntax for adding a container image to a blueprint
+
+[source,toml]
+----
+[[containers]]
+source = "<image_pullspec_with_tag_or_digest>"
+----
+
+.Prerequisites
+
+* You have the exact list of container image references used by the {product-title} version to embed {product-title}'s container images.
```

Comment:
This cannot be a prerequisite, as the following procedure exist to _produce_ that exact list.